### DISCUSSION TITLE
Using "Infinity" vs -1 for infinite repeats

### DISCUSSION THREAD

**Discussion by BrianUribe6** [software-mansion/react-native-reanimated#6551] [Q&A]
Is there any difference between passing the actual [Infinity](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity) constant vs passing `-1` as an argument to `withRepeat` ?

```js

withRepeat(..., Infinity);
// or
withRepeat(..., -1);
```

**Comment by piaskowyk** [comment]
Currently, `Infinity` doesn't work, but I really like this idea! 😃 Would you like to open a PR with this change?

**Reply by BrianUribe6** [reply]
I tried `Infinity` with a couple of animations I have in my project, and it seemed to work. I'm not familiar with the internals of this library, but my assumption is that it is the same as if I said 
```js
const A_REALLY_LARGE_NUMBER = 1 << 31;
withRepeat(..., A_REALLY_LARGE_NUMBER);
```

I'm willing to give it a shot at implementing it, if it really does not work with the existing version, but like I said I'm not familiar with the library, so I might not be right person.
